src/Makefile

```diff
@@ -135,6 +135,9 @@ install: libssu.so ssu faithpd
 	mkdir -p ${PREFIX}/include/unifrac
 	cp task_parameters.hpp ${PREFIX}/include/unifrac/
 	cp api.hpp ${PREFIX}/include/unifrac/
+	cp biom.hpp ${PREFIX}/include/unifrac/
```

Comment:
We need status_enum.hpp, since api.hpp imports it.

Reply:
+1